proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -0,0 +1,183 @@
+syntax = "proto3";
+
+package redpanda.api.console.v1alpha1;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+
+// OIDC Auth settings.
+message OIDCAuth {
+  string token = 1;
+}
```

Comment:
fyi we can probably drop this - rpk doesn't currently support using oidc to authn to a redpanda cluster.

Reply:
I'm going to remove it from the API wiki (https://redpandadata.atlassian.net/wiki/spaces/CORE/pages/804225124/API+Design)

Reply:
Removed. I have left the overall `authentication` as oneof to allow easily adding other options in the future.